### PR TITLE
Update to FileVault Package Maven Plugin 1.1.2

### LINKED ISSUE #954: Update to Filevault Package Maven Plugin 1.1.2
## Feature Request

The most recent version of FileVault introduced validations https://jackrabbit.apache.org/filevault/validation.html. Those are useful to enforce e.g.
correct usage of package types and usage of deprecated resource types and adhering to AEMs content classification.

### PR DESCRIPTION
**Pull request by kwin**
fix validation issues
manage default maven plugin versions
This closes #954

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #954
| Patch: Bug Fix?          |
| Minor: New Feature?      | yes
| Major: Breaking Change?  | no
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes (code comments and or markdown)
| Any Dependency Changes?  | no
| License                  | Apache License, Version 2.0

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -207,14 +218,40 @@
...
                         <group>adobe/cq60</group>
+                        <validatorsSettings>
+                            <jackrabbit-filter>
+                                <options>
+                                    <validRoots>/,/libs,/apps,/apps/core/wcm,/etc,/var,/tmp,/content</validRoots>
+                                </options>
+                            </jackrabbit-filter>
+                            <netcentric-aem-classification>
+                                <!-- <disabled>true</disabled>  -->
+                                <defaultSeverity>WARN</defaultSeverity>
```
Comment (vladbailescu) [inline parent/pom.xml]: Any reason to set this to `WARN` instead of `ERROR`?

File: examples/ui.content/pom.xml [hunk 2]
```diff
@@ -62,9 +62,8 @@
                     </properties>
                     <dependencies>
                         <dependency>
-                            <group>adobe/cq60</group>
-                            <name>core.wcm.components.examples.ui.apps</name>
-                            <version>[${project.version},)</version>
+                            <groupId>com.adobe.cq</groupId>
```
Comment (vladbailescu) [inline examples/ui.content/pom.xml]: maybe also make `examples/ui.apps` dependant on Core Components `content`.

**Comment by vladbailescu** [comment]
Nice PR @kwin ! I'm not extremely kin on whitespace changes and I'm not very sure of `aem-classification-validator` usage (warn vs error) but I'm ok with the changes otherwise.

File: parent/pom.xml (hunk 1 above)
Comment (kwin) [reply parent/pom.xml]: yes, you use some deprecated resource type currently, and I cannot easily resolve that!

**Comment by kwin** [comment]
Reverted the non-deliberate whitespace changes in https://github.com/adobe/aem-core-wcm-components/pull/955/commits/97124f395f4db720502b957265ae29baa742b30d

**Pull request merged**